pyproject.toml

```diff
@@ -86,6 +86,7 @@ click = "^8.1.6"
 spacy-transformers = "^1.2.5"
 boto3 = "^1.28.30"
 botocore = "^1.31.30"
+easyocr = "^1.7.0"
```

Comment:
add `easyocr` for extracting text from image for `test_imafe_redaction` in `test_privateai_scrub.py`